apps/localecash-testing/scripts/generate-auth-roles.sh

```
@@ -0,0 +1,132 @@
+  echo "----------------------------------------"
+}
+
+# Parse remaining arguments as specific roles
+if [ $# -gt 0 ]; then
+  # Validate provided roles against available roles for environment
+  for role in "$@"; do
+    # Normalize role name (capitalize first letter, lowercase rest)
+    normalized_role=$(echo "$role" | awk '{print toupper(substr($0,1,1)) tolower(substr($0,2))}')
+    
+    # Check if role is valid for current environment
+    if [[ " ${AVAILABLE_ROLES[@]} " =~ " ${normalized_role} " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)